topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+Abundances across ranks and submissions were on average predicted better for strain madness than marine data. On the marine data, mOTUs v.2.5.1 had the lowest L1 error while mOTUs v.2.5.1 and MetaPhlAn v.2.9.22 both had the lowest UniFrac error.
+
+On the strain madness data, mOTUs cami1 performed best in L1 norm error whilst MetaPhlAn v.2.9.22 had the lowest UniFrac error.
+
+On the plant-associated data, Bracken v.2.6 had the lowest L1 norm error across ranks and had the lowest UniFrac as “sourmash gather 3.3.2 k31” did.
+
+
+### Compare tools used in CAMI2 with what is already exist in Galaxy
+
+We compared tools submitted for every challenge to understand the start point of CAMI2 reproduction in the Galaxy. We collected metadata for each tool:
+
+* source code, 
+* Publication that describes the tool
+* Tool version
+* Availability in Galaxy
+* owner/ developer in Galaxy	
+* Galaxy wrapper directory	
+* Tool version in Galaxy
+* Relevance in Galaxy	
+* Availability in conda-forge / bioconda	
+* Notes on their performance	
+* Ranking according to CAMI2 results	
+
+> ### {% icon details %} CAMI2 Assembly tools comparison
+>
+> <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml?gid=0&amp;single=true&amp;widget=true&amp;headers=false" width="900" height="600" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+>
+{: .details}
+
+> ### {% icon details %} CAMI2 Genome binning tools comparison
+>
+> <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml?gid=836882193&amp;single=true&amp;widget=true&amp;headers=false" width="900" height="600" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+>
+{: .details}
+
+> ### {% icon details %} CAMI2 Taconomic binning tools comparison
+>
+> <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml?gid=794114188&amp;single=true&amp;widget=true&amp;headers=false" width="900" height="600" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+>
+{: .details}
+
+> ### {% icon details %} CAMI2 Taxonomic profiling tools comparison
+>
+> <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml?gid=2127016351&amp;single=true&amp;widget=true&amp;headers=false" width="900" height="600" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+>
+{: .details}
```

Comment:
Linking all of the spreadsheets via iframes feels a bit odd, I think it might be a bit nicer to collect all of those in a details box as links, something like this instead of the iframes. Would that be ok for you? (note, links aren't correct here.)

```
> ### {% icon details %} CAMI2 Details
>
> - [CAMI2 Assembly tools comparison](https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml)
> - [CAMI2 Genome binning tools comparison](https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml)
> - [CAMI2 Taconomic binning tools comparison](https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml)
{: .details}
```